packages/server-core/src/user/avatar/avatar.hooks.ts

```diff
@@ -164,7 +164,8 @@ export default {
     all: [() => schemaHooks.validateQuery(avatarQueryValidator), schemaHooks.resolveQuery(avatarQueryResolver)],
     find: [
       iffElse(isAction('admin'), verifyScope('admin', 'admin'), ensureUserAccessibleAvatars),
-      discardQuery('action')
+      discardQuery('action'),
+      iff((context) => context.params.query?.$sort?.['user'], discardQuery('$sort.user'))
```

Comment:
Need to do user name sorting on server to do it on all records

Reply:
implemented [`sortByUsername`](https://github.com/EtherealEngine/etherealengine/pull/9073/commits/15ae71e5d6c36df0374bc6569dcf64a458fee84c#diff-5d8f220f6a127251c7b3c898887efb63a8dff4dfd9d9daab035d926c1ecb48d0R101-R117_